api/types.go

```diff
@@ -161,6 +166,36 @@ type (
 		CallbackURL string   `json:"callbackURL"`
 		Scopes      []string `json:"scopes"`
 	}
+
+	// AccountResp is the response body for the [GET] /tpool/fee endpoint
+	TPoolResp struct {
+		RecommendedFee types.Currency
+	}
+
+	// AccountResp is the response body for the [GET] /accounts endpoint
+	AccountResp []accounts.Account
+
+	// VolumeResp is the response body for the [GET] /volumes endpoint
+	VolumeResp []storage.VolumeMeta
+
+	// AlertResp is the response body for the [GET] /alerts endpoint
+	AlertResp []alerts.Alert
+
+	// PeerResp is the response body for the [GET] /syncer/address endpoint
+	PeerResp []Peer
+
+	// SyncAddrResp is the response body for the [GET] /syncer/peers endpoint
+	SyncAddrResp modules.NetAddress
+
+	// WalletTransactionsResp is the response body for the [GET] /wallet/transactions endpoint
+	// WalletTransactionsResp is the response body for the [GET] /wallet/pending endpoint
+	WalletTransactionsResp struct {
```

Comment:
This will change the /wallet/transactions response. You should make this like
`PeerResp` and have separate types for `/wallet/transactions` and `/wallet/pending`.

Reply:
fixed in next commit